src/include/souffle/SouffleInterface.h

```diff
@@ -61,6 +61,9 @@ class Relation {
          * Required for identifying type of iterator
```

Comment:
Indeed - this is legacy and should be removed.

Reply:
Will fix.